### ISSUE TITLE
Dark Theme display issue: banded table rows incorrect background color

### ISSUE THREAD

**Issue by darianmiller** [ideasawakened/UniversalMarkupEditor#11]
The banded table rows do not correctly display the secondary background color when the Dark Theme is active

This is a snippet of the help page exported to browser:
![image](https://user-images.githubusercontent.com/3165264/75810328-002df480-5d50-11ea-85e0-1cfde71f2654.png)

**Comment by darianmiller** [comment]
We use CSS from https://github.com/sindresorhus/github-markdown-css which is the basis for the "light" theme. The dark theme is an edited version and the text color of this CSS element was missed.

before
````
.markdown-body table tr:nth-child(2n) {
  background-color: #DADADA;
}
````
after
````
.markdown-body table tr:nth-child(2n) {
  color: #000000;
  background-color: #DADADA;
}
````

**Comment by darianmiller** [comment]
Will be fixed in release 1.1
